apps/web/src/modules/dao/components/DaoCard/DaoCard.tsx

```diff
@@ -50,9 +52,10 @@ export const DaoCard = ({
   if (!isMounted) return null
 
   const isOver = !!endTime ? dayjs.unix(Date.now() / 1000) >= dayjs.unix(endTime) : true
+  const chain = PUBLIC_DEFAULT_CHAINS.find((x) => x.id === chainId)
```

Comment:
Again, I'd opt to memoize. Not a huge deal. Not a huge cost either though.

Reply:
Same comment as the other memoization one